### PR TITLE
update load config with types

### PR DESCRIPTION
**Pull request by jchip**

### CODE REVIEW

**Review by divyakarippath — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -17,11 +17,11 @@ const confippet = {
   providerTypes: providerTypes,
   extHandlers: extHandlers,
   util: util,
-  loadConfig: loadConfig
+  loadConfig: loadConfig,
 };
 
 Object.defineProperty(confippet, "config", {
-  get: () => require("../config")
+  get: () => require("../config"),
```
Comment (vinaya-thimmappa) [inline src/index.ts:24]: why is this change required? it is breaking my build.since there is no config inside the src directory

Comment (vinaya-thimmappa) [reply src/index.ts:24]: Error: Cannot find module '../config'
Require stack:
- node_modules/electrode-confippet/lib/index.js

getting this error

Comment (jchip) [reply src/index.ts:24]: publish config issue.  fixed in 1.7.1, thanks.
